roofit/roofitcore/inc/RooAbsData.h

```diff
@@ -55,8 +55,10 @@ class RooAbsData : public TNamed, public RooPrintable {
   RooAbsDataStore* store() { return _dstore ; }
   const RooAbsDataStore* store() const { return _dstore ; }
   const TTree* tree() const ;
+  TTree* export_tree() const ;
```

Comment:
This method name is not following the ROOT naming conventions. I'd opt for `GetClonedTree` but this is up to @lmoneta.